src/backend/storeManagers/nile/games.ts

```diff
@@ -0,0 +1,591 @@
+    }
+    return { status: 'error', error: res.error }
+  }
+  addShortcuts(appName)
+  installState(appName, true)
+  const metadata = getInstallMetadata(appName)
+  await setup(appName, metadata?.path)
+
+  return { status: 'done' }
+}
+
+export function isNative(): boolean {
+  return isWindows
+}
+
+/**
+ * Adds a desktop shortcut to $HOME/Desktop and to /usr/share/applications
+ * so that the game can be opened from the start menu and the desktop folder.
+ * Both can be disabled with addDesktopShortcuts and addStartMenuShortcuts
+ * @async
+ * @public
+ */
+export async function addShortcuts(appName: string, fromMenu?: boolean) {
+  return addShortcutsUtil(getGameInfo(appName), fromMenu)
+}
+
+/**
+ * Removes a desktop shortcut from $HOME/Desktop and to $HOME/.local/share/applications
+ * @async
+ * @public
+ */
+export async function removeShortcuts(appName: string) {
+  return removeShortcutsUtil(getGameInfo(appName))
+}
+
+export async function launch(
+  appName: string,
+  launchArguments?: string
+): Promise<boolean> {
+  const gameSettings = await getSettings(appName)
+  const gameInfo = getGameInfo(appName)
+
+  const {
+    success: launchPrepSuccess,
+    failureReason: launchPrepFailReason,
+    rpcClient,
+    mangoHudCommand,
+    gameModeBin,
+    steamRuntime
+  } = await prepareLaunch(gameSettings, gameInfo, isNative())
+
+  if (!launchPrepSuccess) {
+    appendFileSync(
+      logFileLocation(appName),
+      `Launch aborted: ${launchPrepFailReason}`
+    )
+    showDialogBoxModalAuto({
+      title: t('box.error.launchAborted', 'Launch aborted'),
+      message: launchPrepFailReason!,
+      type: 'ERROR'
+    })
+    return false
+  }
+  const exeOverrideFlag = gameSettings.targetExe
+    ? ['--override-exe', gameSettings.targetExe]
+    : []
+
+  let commandEnv = isWindows
+    ? process.env
+    : { ...process.env, ...setupEnvVars(gameSettings) }
+  let wineFlag: string[] = []
+
+  if (!isNative()) {
+    // -> We're using Wine/Proton on Linux or CX on Mac
+    const {
+      success: wineLaunchPrepSuccess,
+      failureReason: wineLaunchPrepFailReason,
+      envVars: wineEnvVars
+    } = await prepareWineLaunch('nile', appName)
+    if (!wineLaunchPrepSuccess) {
+      appendFileSync(
+        logFileLocation(appName),
+        `Launch aborted: ${wineLaunchPrepFailReason}`
+      )
+      if (wineLaunchPrepFailReason) {
+        showDialogBoxModalAuto({
+          title: t('box.error.launchAborted', 'Launch aborted'),
+          message: wineLaunchPrepFailReason,
+          type: 'ERROR'
+        })
+      }
+      return false
+    }
+
+    commandEnv = {
+      ...commandEnv,
+      ...wineEnvVars
+    }
+
+    const { bin: wineExec, type: wineType } = gameSettings.wineVersion
+
+    // Fix for people with old config
+    const wineBin =
+      wineExec.startsWith("'") && wineExec.endsWith("'")
+        ? wineExec.replaceAll("'", '')
+        : wineExec
```

Comment:
I think this part is not needed, as there is no old config for Amazon games

Reply:
Are we sure that global configs can't trigger this case?

Reply:
hmm, maybe you're right